docker-compose.yml

```diff
@@ -21,4 +22,12 @@ services:
       - OCTOBOX_DATABASE_USER=postgres
       - OCTOBOX_DATABASE_PASSWORD=development
       - OCTOBOX_DATABASE_HOST=database.service.octobox.internal
-    command: bin/docker-start
+    networks:
+      - internal
+
+volumes:
+  pg_data:
+
+networks:
+  internal:
+    driver: bridge            
```

Comment:
why do we networks here?

Reply:
ohh it to talk to postgres

Reply:
links is going to be deprecated soon, https://docs.docker.com/engine/userguide/networking/default_network/dockerlinks/ so the idea is that containers on the same network can communicate directly, as am not a Ruby developer I couldn't figure out why the main container can't communicate with the database even though you can ping it correctly.

Reply:
I was able to remove the links completely so now we only depend on the networks, but we do need to make sure that the database container will be named correctly.